core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -874,6 +874,26 @@
    */
   public static final String CARBON_INSERT_PERSIST_ENABLED_DEFAULT = "false";
 
+  /**
+   * Which storage level to persist dataset when insert into data
+   * with 'carbon.insert.persist.enable'='true'
+   */
+  @CarbonProperty
+  public static final String CARBON_INSERT_STORAGE_LEVEL =
+      "carbon.insert.storage.level";
```

Comment:
please add it to document also: https://github.com/apache/carbondata/blob/master/docs/configuration-parameters.md

Reply:
Done